airbyte-integrations/connectors/destination-postgres/build.gradle

```diff
@@ -72,6 +73,7 @@ configurations {
 }
 
 dependencies {
+    implementation("org.jooq:jooq:$jooqVersion")
```

Comment:
Why do we need Jooq, that looks sus

Reply:
Don't we have a way to have the SqlDataType from the posstgres drive?

Reply:
I guess this was my way of using valid sql types when doing the mapping in`toDialectType`, I think I did this because I saw it on other parts of the codebase. But now that you call this out you are right, I can just map them to the string values without jooq

Reply:
hm I don't know if I want to change the driver just for this. I think I'm just going to add an enum with the types.